### ISSUE TITLE
init.meta.json alongside init.server.lua errors

### ISSUE THREAD

**Issue by tacheometry** [rojo-rbx/rojo#517]
Having the following file structure:
* `folder`
    * `init.meta.json`
    * `init.server.lua`

or:

* `folder`
    * `init.meta.json`
    * `init.client.lua`

With the `init.meta.json`'s contents being:

```json
{
    "properties": {
        "Disabled": true
    }
}
```
Does not produce the expected results when running `rojo build`, giving the following error:
```
$ rojo build --output Test.rbxl
[ERROR rojo] error applying meta file RojoRepro/code/init.meta.json

        Caused by:
            Unknown property Folder.Disabled
```
Minimal reproduction project: [RojoRepro.zip](https://github.com/rojo-rbx/rojo/files/7842814/RojoRepro.zip)

**Comment by unix-system** [comment]
Identified issue.

https://github.com/rojo-rbx/rojo/blob/master/src/snapshot_middleware/lua.rs#L69 runs a snapshot to test if the class_name outcome is Folder or not. However, `dir_snapshot` itself will import metadata via the `meta_file` impl. To solve this, only the class_name should be checked.

**Comment by tacheometry** [comment]
Resolved in v7.2.0
